robottelo/ui/locators.py

```diff
@@ -672,6 +672,13 @@ def __iter__(self):
     "entity_checkbox": (
         By.XPATH,
         "//label[normalize-space(.)='%s']/input[@type='checkbox']"),
+    "entity_select_list": (
+        By.XPATH,
+        "//ul/li/div[normalize-space(.)='%s']"),
```

Comment:
I'd be as much specific as possible here in locating the correct element => filter the `ul` element by class: 
```
"//ul[@class='select2-results']/li/div[normalize-space(.)='%s']"
```
. This will prevent any other accidental `<ul><li><div>` being matched.